### ISSUE TITLE
About pre-trained model

### ISSUE THREAD

**Issue by kaiyi98** [shaofengzeng/SuperPoint-Pytorch#26]
Thank your great work!
The superpoint_bn.pth and superpoint_v1.pth provided by you have some difference? I want the magicpoint parameter trained from synthetic dataset , which I can train in my dataset rather COCO. Could you share it ?
Thanks!

**Comment by shaofengzeng** [comment]
Hi, both superpoint_bn.pth and superpoint_v1.pth are superpoint model, not magicpoint model. Training magicpoint is much easier than superpoint, have a try!

**Comment by kaiyi98** [comment]
感谢回复，我用您的代码训练第一步，人造图形训练magicpoint，怎么到20个epoch左右验证损失就收敛了，但是原论文说magicpoint训练了200000个iteration，然后给coco打点，打出来的点效果并不理想。还望解答，谢谢。

**Comment by RomanticFisher** [comment]
> 感谢回复，我用你的代码训练第一步，人造图形训练magicpoint，怎么到20个epoch左右验证就收敛了，但是原论文说magicpoint训练了200000个迭代，给coco打点，打出来的点效果不太理想。还望解答，谢谢。

I also tried using magipoint to generate keypoints. The loss has converged, but there is no way to generate reliable keypoints for them in actual images. Even if the threshold is changed to 0.001, it has no effect. If you find any problems, please let me know. Thank you.

**Comment by Alexander8848** [comment]
> 感谢回复，我用你的代码训练第一步，人造图形训练magicpoint，怎么到20个epoch验证就收敛了，但是原论文说magicpoint训练了200000个迭代，然后给coco打点，打出来的点效果理想并不理想。还望解答，谢谢。

请问您解决这个问题了吗
